src/main/org/deidentifier/arx/criteria/EDDifferentialPrivacy.java

```diff
@@ -160,62 +160,59 @@ public int getRequirements(){
         return ARXConfiguration.REQUIREMENT_COUNTER |
                ARXConfiguration.REQUIREMENT_SECONDARY_COUNTER;
     }
-
-    /**
-     * Creates a random sample based on beta
-     *
-     * @param manager
-     */
+    
+    @Override
     public void initialize(DataManager manager, ARXConfiguration config){
         
-        // Needed for consistent de-serialization. We need to call this
-        // method in the constructor of the class DataManager. The following
-        // condition should hold, when this constructor is called during 
-        // de-serialization, when we must not change the subset.
-        if (subset != null && this.manager == null) {
-            this.manager = manager;
+        if (config != null) {

```

Comment:
Because  this is the only privacy model that dynamically constructs a subset, we need to call this method twice. This is the second call and can be ignored.